CONTRIBUTING.md

```diff
@@ -28,27 +28,20 @@ We use `pytest` for our unit testing, making use of `parametrized` to inject cas
 
 These are planned once the SDK has been stabilized and a Flagd provider implemented. At that point, we will utilize the [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features/evaluation.feature) to validate against a live, seeded Flagd instance.
 
-### Packaging
```

Comment:
I didn't feel like we needed to maintain this list in the contrib doc. It will likely get out of date and provide minimal value.